modules/mixed-arch-import-imagestreams.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-import-imagestreams_{context}"]
+
+= Importing manifest lists on your heterogeneous cluster 
+
+On a {product-title} {product-version} heterogeneous cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option in order to import the manifest list.
+
+.Prerequisites 
+
+* You installed the Openshift CLI (`oc`).
+
+.Procedure 
+
+* You can change the default `importMode` option by running the following command: 
+
```

Comment:
reword suggestion: "You can change the default `importMode` option for an imagestream... cli-artifacts is shown here as an example"

Reply:
Is that the right context for the cli artifacts?

Reply:
you did..i was just saying maybe we should call out that the example used here is cli-artifacts...or you could change that to `tools` since i have the output for what the tools istag would look like when imported